feature_server/run.py

```diff
@@ -57,6 +57,10 @@ def choose_path(base,top):
 
 # add it to the path so we can import scripts
 sys.path.append(config_dir)
+# add our package to path too so scripts can import `feauture_server/`
```

Comment:
better make sure none of the files then conflict with other packages/modules then. I'm especially worried about `map.py`, `cfg.py`, `run.py`....
I think it could be safer to keep the namespacing.

You can always import like this still:
```
from pysnip import commands, console, irc
print commands
```